pom.xml

```diff
@@ -33,7 +33,7 @@
     <api-security-java.version>0.2.10</api-security-java.version>
     <commons-io.version>2.4</commons-io.version>
 
-    <spring-boot-dependencies.version>2.1.3.RELEASE</spring-boot-dependencies.version>
+    <spring-boot-dependencies.version>2.1.10.RELEASE</spring-boot-dependencies.version>
```

Comment:
Do you think we should also update the spring boot maven plugin version as well to keep inline with the spring boot dependency version upgrade ?

Reply:
Updated spring boot maven version to match the dependencies version